stable/spring-cloud-data-flow/README.md

```diff
@@ -30,6 +30,13 @@ If you are using a cluster that does not have a load balancer (like Minikube) th
 $ helm install --name my-release --set server.service.type=NodePort stable/spring-cloud-data-flow
 ```
 
+To restrict the load balancer to an IP address range:
+
+```bash
+$ helm install --name my-release  --set server.service.loadBalancerSourceRanges='[10.0.0.0/8]' stable/spring-cloud-data-flow
```

Comment:
the `Data Flow Server Configuration` section should also be updated with a row for this property

Reply:
Documentation added.